charts/fluent-bit/values.yaml

```diff
@@ -11,7 +11,12 @@ image:
   pullPolicy: Always
   # tag:
 
+testImage:
+  repository: busybox
+  tag: latest
+
 imagePullSecrets: []
+  # - name: mysecret
```

Comment:
```suggestion
testFramework:
  image: busybox
  tag: latest

imagePullSecrets: []
```

nit: Kindly nest in `testFramework` as this is a pattern seen in other charts and allows for future test related config to be neatly added